static/js/main.js

```diff
@@ -338,7 +356,10 @@ function takeVideoScreenshot(query, xPercent, yPercent, widthPercent, heightPerc
 
     let id = randomHex();
 
-    let index_string = String(Object.keys(window.shark.screenshots).length).padStart(3, '0');
+    // Get screenshot counter value and update counter.
+    let index_string = String(window.shark.screenshot_counter).padStart(3, '0');
+    window.shark.screenshot_counter ++;
```

Comment:
`++` should not have a space before it.
(This is generally true with all unary operators, e.g., `!`.)

(This is a pretty wild style. Have you been taught or seen it before?)

Reply:
Never been told about ++ spacing formally but it makes sense given it's a unary operator and I'd never put a space between other unary operators like !.